WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -130,28 +130,24 @@ - (void)viewDidLoad
 
 - (void)setupHeaderView
 {
-    self.headerView = [[UIView alloc] initWithFrame:CGRectMake(0.0, 0.0, CGRectGetWidth(self.view.bounds), BLVCHeaderViewHeight)];
-
-    UILabel *label = [[UILabel alloc] initWithFrame:CGRectZero];
-    label.translatesAutoresizingMaskIntoConstraints = NO;
-    label.numberOfLines = 0;
-    label.textAlignment = NSTextAlignmentCenter;
-    label.textColor = [WPStyleGuide allTAllShadeGrey];
-    label.font = [WPFontManager openSansRegularFontOfSize:14.0];
-    label.text = NSLocalizedString(@"Select which sites will be shown in the site picker.", @"Blog list page edit mode header label");
-    [self.headerView addSubview:label];
-
-    // Layout
-    NSDictionary *views = NSDictionaryOfVariableBindings(label);
-    NSDictionary *metrics = @{@"padding": @(BLVCHeaderViewLabelPadding)};
-    [self.headerView addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"|-(padding)-[label]-(padding)-|"
-                                                                          options:0
-                                                                          metrics:metrics
-                                                                            views:views]];
-    [self.headerView addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-(padding)-[label]-(padding)-|"
-                                                                          options:0
-                                                                          metrics:metrics
-                                                                            views:views]];
+    self.headerView = [[UIView alloc] initWithFrame:CGRectZero];
+
+    self.headerLabel = [[UILabel alloc] initWithFrame:CGRectZero];
+    self.headerLabel.numberOfLines = 0;
+    self.headerLabel.textAlignment = NSTextAlignmentCenter;
+    self.headerLabel.textColor = [WPStyleGuide allTAllShadeGrey];
+    self.headerLabel.font = [WPFontManager openSansRegularFontOfSize:14.0];
+    self.headerLabel.text = NSLocalizedString(@"Select which sites will be shown in the site picker.", @"Blog list page edit mode header label");
+    [self.headerView addSubview:self.headerLabel];
+}
+
+- (void)updateHeaderSize
+{
+    float labelWidth = CGRectGetWidth(self.view.bounds) - 2 * BLVCHeaderViewLabelPadding;
```

Comment:
A bit nitpicky, but maybe declare this with the foundation type CGFloat instead of the C type. It gives us a bit of future type safety.